code_examples/sdk_examples/src/workshop/attester/generateDid.ts

```diff
@@ -1,51 +1,38 @@
-import { config as envConfig } from 'dotenv'
-
 import * as Kilt from '@kiltprotocol/sdk-js'
-
+import { config as envConfig } from 'dotenv'
 import { generateAccount } from './generateAccount'
-import { generateKeypairs } from './generateKeypairs'
 
 export async function createFullDid(
-  submitterAccount: Kilt.KiltKeyringPair
+  creatorAccount: Kilt.KiltKeyringPair & { type: 'ed25519' }
```

Comment:
The more correct (because more inclusive) typing would be the following, which we also use in a few other locations in the docs:
```suggestion
  creatorAccount: Kilt.KiltKeyringPair & { type: 'ed25519' | 'sr25519' | 'ecdsa' }
```

Reply:
This is the complete set of key types that can be handled by the createFromAccount and setAttestationKey transactions.